managers/CpbStoryManager.cpp

```diff
@@ -11,31 +11,35 @@ StoryManager::StoryManager(QObject *parent) :
 {
 }
 
-void StoryManager::createStory(Sprint* sprint)
+void StoryManager::createStory(Sprint* sprint, QString storyName = "")
```

Comment:
1) swap it and remove default "", i.e. (const String& storyName, Sprint* sprint)
2) const String&